skin.css

```diff
@@ -1,13 +1,42 @@
 .highlight {
-    background-color: #ddd;
+	background-color: #ddd;
 }
 
 button.btn {
-  border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
-  font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+	margin-left: 5px;
+	padding: 10px 65px;
+	font-size: 18px;
+	line-height: 1.5;
+	font-weight: normal;
+	text-align: center;

```

Comment:
This declaration isn't necessary, so while it doesn't hurt it is better to remove it to make it clear what the ruleset does. It isn't necessary because the `text-align` attribute doesn't apply to block elements - see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align).